src/Functions/LowerUpperUTF8Impl.h

```diff
@@ -37,40 +41,70 @@ struct LowerUpperUTF8Impl
             return;
         }
 
-        res_data.resize(data.size());
-        res_offsets.resize_exact(offsets.size());
 
-        String output;
+        UErrorCode error_code = U_ZERO_ERROR;
```

Comment:
In l. 34, I think we should rather check for `input_rows_count == 0` instead of `data.empty()`.